tests/plugins/environments/torch_elastic_deadlock.py

```diff
@@ -0,0 +1,37 @@
+import os
```

Comment:
nit on test location: this isn't really torchelastic specific
it's more about how the DDP plugin handles deadlocks when processes are created externally to the plugin
maybe we could add it here: under `test_ddp_plugin_deadlock_detection` ? 
https://github.com/PyTorchLightning/pytorch-lightning/tree/master/tests/plugins

Reply:
I will make another PR with this update. GPU CI has been flicky today + I want to give a try to multi node and torch elastic.